src/libraries/Microsoft.PowerFx.Core/Public/Engine.cs

```diff
@@ -178,12 +178,17 @@ private protected virtual IIntellisense CreateIntellisense()
             return IntellisenseProvider.GetIntellisense(Config);
         }
 
+        public IIntellisenseResult Suggest(string expression, RecordType parameterType, int cursorPosition)
+        {
+            return Suggest(expression, parameterType, cursorPosition, null);
+        }
+
         /// <summary>
-        /// Get intellisense from the formula.
+        /// Get intellisense from the formula, with parser options.
         /// </summary>
-        public IIntellisenseResult Suggest(string expression, RecordType parameterType, int cursorPosition)
+        public IIntellisenseResult Suggest(string expression, RecordType parameterType, int cursorPosition, ParserOptions options)
         {
-            var result = Check(expression, parameterType);
+            var result = Check(expression, parameterType, options);
             var binding = result._binding;
             var formula = new Formula(expression, Config.CultureInfo);
```

Comment:
>Config.CultureInfo [](http://example.com/codeflow?start=50&length=18)

Future thing - Wondering if this needs to be flexible too. Maybe we get it from CheckResult ?

Reply:
AFAIR, the culture is defined at the config level, not per expression/check/eval